app/models/host.rb

```diff
@@ -22,7 +22,43 @@ class Host < ApplicationRecord
   validates :name, presence: true
   validates :account, presence: true
 
+  def in_inventory
+    ::HostInventoryAPI.new(
+      User.current.account,
+      ::Settings.host_inventory_url,
+      nil
+    ).inventory_host(id)
+  end
```

Comment:
:+1: This seems really useful, and I like how it looks `Host.first.in_inventory`